_includes/utilities/values-intro.html

```diff
@@ -1,5 +1,5 @@
 <h3 class="utilities-section-title">Custom values</h3>
-<p class="utility-text">By default, USWDS utilities output the standard values listed in the documentation above. Customize and refine any utility family’s specific values by editing the <code>$[utility_family]-manual-values</code> and <code>$[utility_family]-palettes</code> variables in  <code>_uswds-theme-utilities.scss</code>.</p>
+<p class="utility-text">By default, USWDS utilities output the standard values listed in the documentation above. Customize and refine any utility family’s specific values by editing the <code>$[utility_family]-manual-values</code> and <code>$[utility_family]-palettes</code> variables in  <code>_uswds-theme.scss</code>.</p>
```

Comment:
Instead of saying "above," (which can be tough to understand exactly where above, for people using assistive devices OR if there's just a lot of documentation on the page, regardless) can you link to the specific area? So, for example, "values listed in the [standard output table](sample.url.com)"

Reply:
This section is imported into multiple different utilities pages so I'm not sure if there is a way to specifically link to each table correctly. I tried linking to the common `#utilities-section` that each section uses but it didn't work as expected unfortunately. @mejiaj You have any idea how we could make that work?

Reply:
Agree with @bonnieAcameron we should avoid saying "above." We should find alternative language **or** point to the specific sections.

@mahoneycm this is an include, so it _could_ take parameters. You can learn more about that in the Jekyll [documentation](https://jekyllrb.com/docs/includes/#passing-parameters-to-includes).